configs/HPECray-EPYC3-MI250X-Slingshot/system_definition.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: HPECray-EPYC3-MI250X-Slingshot
+  manufacturer:
+    vendor: HPE-Cray
+    name: EX235a
+  processor:
+    vendor: AMD
+    name: EPYC-Zen3
+    ISA: x86_64
+  accelerator:
+    vendor: AMD
+    name: MI250X
+    GPU_ISA: gfx90a
```

Comment:
Is `GPU_ISA` the right word for these? I would've thought something like `architecture`.

Reply:
Please see slack for discussion.